doc/cephfs/fs-nfs-exports.rst

```diff
@@ -0,0 +1,58 @@
+=======================
+CephFS Exports over NFS
+=======================
+
+CephFS namespaces can be exported over NFS protocol using the
+`NFS-Ganesha NFS server <https://github.com/nfs-ganesha/nfs-ganesha/wiki>`_.
+
+Requirements
+============
+
+-  Latest Ceph file system with mgr and dashboard enabled
+-  'nfs-ganesha', 'nfs-ganesha-ceph' and nfs-ganesha-rados-grace packages
+   (version 2.7.6-2 and above)
+
+Create NFS Ganesha Cluster
+==========================
+
+.. code:: bash
+
+    $ ceph fs nfs cluster create [--size=1] <cluster_id>
```

Comment:
Just curious: [rgw-nfs](https://docs.ceph.com/docs/master/radosgw/nfs/) is totally unrelated and managing this would require something completely new? Otherwise we might be able to simply use `ceph nfs` instead of `ceph fs nfs`

Reply:
Not sure about it, I will have to check.

Reply:
> Just curious: [rgw-nfs](https://docs.ceph.com/docs/master/radosgw/nfs/) is totally unrelated and managing this would require something completely new? Otherwise we might be able to simply use `ceph nfs` instead of `ceph fs nfs`

Not sure about it, I will have to check.

Reply:
I like this idea. There is no reason you can't export both rgw and cephfs at the same time from the same daemons, and it probably should be treated as common infrastructure that way.

It's just a different kind of export (FSAL_RGW vs FSAL_CEPH). RGW is simpler and less stateful than cephfs. Some of the clustered recovery isn't really needed for it, but it shouldn't hurt anything either.

Reply:
There is a caveat to my previous comment on this thread. The recommended configurations for FSAL_CEPH and FSAL_RGW are different in global settings. We'd probably need to consider how to fix that in ganesha first, so until then we probably do want separate infrastructure for FSAL_CEPH and FSAL_RGW exports.